Test-№2.0-from-23.11.2022/task1.c

```diff
@@ -0,0 +1,250 @@
+#include <stdio.h>
+#include <stdlib.h>
+#include <stdbool.h>
+#define dateSize 10
+
+bool isItANumber(char *queue, int iterator)
+{
+    char numbers[10] = {'0', '1', '2', '3', '4', '5', '6', '7', '8', '9'};
+    bool answer = false;
+    for (int i = 0; i < dateSize; ++i)
+    {
+        if (queue[iterator] == numbers[i])
+        {
+            answer = true;
+            break;
+        }
+    }
+    return answer;
+}
+
+bool isItADot(char *queue, int iterator)
+{
+    if (queue[iterator] == '.')
+    {
+        return true;
+    }
+    return false;

```

Comment:
```suggestion
    return queue[iterator] == '.';
```
Лишний тавтологичный условный оператор. 
(-0.5 балла)